src/router/Router.jsx

```diff
@@ -8,17 +8,19 @@ import FAQ from "../pages/FAQ";
 import PrivateRoute from "./PrivateRoute";
 import Login from "../pages/Login";
 import { AuthContextProvider } from "../context/AuthContext";
+import PageNotFound from "../pages/PageNotFound/PageNotFound";
 
 const Router = () => (
   <AuthContextProvider>
     <BrowserRouter basename="/HacktoberFest">
       <Navigation />
       <Switch>
         <Route path="/" exact component={Home} />
-        <Route path="/information" component={Information} />
-        <Route path="/faq" component={FAQ} />
-        <Route path="/login" component={Login} />
-        <PrivateRoute path="/profile" component={Profile} />
+        <Route path="/information" exact component={Information} />
+        <Route path="/faq" exact component={FAQ} />
+        <Route path="/login" exact component={Login} />
+        <PrivateRoute path="/profile" exact component={Profile} />
```

Comment:
🔧  I think we only want exact on the `/` path.

Reply:
Addressed: d612efc22a699d66c1ab524fa76d86e4b6c65096